scripts/redeploy-ui.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Is this for redeploying UI locally? You can run `pnpm run build:watch` in `addons/addon-base-raas-ui/packages/base-raas-ui` and keep the terminal up to get local changes reflected in local deployment in real time.

Reply:
It's actually for redeploying the UI non-locally, since at the time I wasn't using the local environment. I can remove this file, I had planned to remove it prior to merge to mainline anyway.